spec/views/staff/shared/activities/activity_spec.rb

```diff
@@ -245,7 +245,10 @@
 
   RSpec::Matchers.define :show_basic_details do
     match do |actual|
-      expect(rendered).to have_content activity_presenter.partner_organisation_identifier
+      if activity_presenter.is_project? || activity_presenter.partner_organisation_identifier.present?
+        expect(rendered).to have_content activity_presenter.partner_organisation_identifier
+      end
```

Comment:
Perhaps we should abstract `activity_presenter.is_project? || activity_presenter.partner_organisation_identifier.present?` into a method (e.g. `partner_organisation_identifier_visible?`) on the model or the presenter, since it's used three times now - here and in the two partials in `app/views/staff/shared/activities`